server.elm

```diff
@@ -2793,6 +2809,12 @@ editionscript = """
           e.preventDefault();
           if(!redo()) popupMessage("Nothing to redo!");
         }
+        if(editor_model.linkSelectMode) {
+          if(e.which == 27) {
```

Comment:
Can you 1) Explain in a comment what key is this 27 is for 2) Explain this mechanism in some documentation, such as the editor's README.md ?